pom.xml

```diff
@@ -417,18 +363,32 @@
                 <artifactId>maven-resources-plugin</artifactId>
                 <version>2.7</version>
             </plugin>
-	    <plugin>
-	      <!-- This plugin will set properties values using dependency information -->
-	      <groupId>org.apache.maven.plugins</groupId>
-	      <artifactId>maven-dependency-plugin</artifactId>
-	      <executions>
-		<execution>
-		  <goals>
-		    <goal>properties</goal>
-		  </goals>
-		</execution>
-	      </executions>
-	    </plugin>
+            <plugin>
```

Comment:
In general I approve of changing tabs to spaces.  However:
 * If you do that, please do so in a separate pull request, to make clear the substantive changes in this pull request.
 * Don't make formatting improvements in a fork of an upstream repository, because it makes the diffs unnecessarily large.

Reply:
I'm a little confused.  Point (1) is a general comment, but point (2) says don't do it in this case?

Reply:
Your understanding is correct.  Sorry for the confusion!  I think I already undid the whitespace changes.